<launch>
  <!-- Wavemap params -->
  <arg name="param_file"
       default="$(find wavemap_ros)/config/ouster_os0.yaml"
       doc="Name of the file from which to read wavemap's server params."/>

  <!-- Dataset params -->
  <arg name="only_first_rosbag"
       default="false"
       doc="If true, replay only the first the rosbag in the dataset. If false, replay everything."/>

  <!-- Select the rosbag paths -->
  <!-- NOTE: Relative paths will be resolved w.r.t. ROS_HOME (default ~/.ros) -->
  <arg if="$(arg only_first_rosbag)" name="rosbag_path"
       value="../data/newer_college/2021-ouster-os0-128-alphasense/collection3/easy/nc-easy-undistorted_0.bag"/>
  <arg unless="$(arg only_first_rosbag)" name="rosbag_path" value="
../data/wavemap_experiments/newer_college/rosbags/math/nc-easy-undistorted_0.bag
../data/wavemap_experiments/newer_college/rosbags/math/nc-easy-undistorted_1.bag"/>

  <node pkg="tf"
        type="static_transform_publisher"
        name="static_tf_map_to_odom"
        args="-23.7154 -31.2451 1.02503 -0.00587066 0.00356705 0.841367 0.54042
              map odom 100"/>


  <!-- Run wavemap -->
  <include file="$(find wavemap_ros)/launch/datasets/dataset_base.launch"
           pass_all_args="true"/>
</launch>
